src/elli/dispersions/base_dispersion.py

```diff
@@ -224,7 +224,7 @@ def __init__(self, *disps: Dispersion) -> None:
         self.dispersions = disps
 
     def dielectric_function(self, lbda: npt.ArrayLike) -> npt.NDArray:
-        dielectric_function = np.sum(
+        dielectric_function = sum(
```

Comment:
Why not use numpy's sum? Should be faster as these are all numpy arrays, right?

Reply:
I see. I wasn't aware of this. I don't know if the numpy version gives so much benefit over just using sum here, so probably it is not worth the effort of digging deeper.